scipy/_lib/tests/test__util.py

```diff
@@ -19,9 +19,17 @@
                               rng_integers, _validate_int, _rename_parameter,
                               _contains_nan, _rng_html_rewrite, _lazywhere)
 import scipy._lib.array_api_extra as xpx
+from scipy.conftest import lazy_jax_jit
 from scipy import cluster, interpolate, linalg, optimize, sparse, spatial, stats
 
 
+lazy_jax_jit(_contains_nan,
+             static_argnums=(1, ),
+             static_argnames=("nan_policy", "xp_omit_okay", "xp"))
+# FIXME @jax.jit fails: complex bool mask
+# lazy_jax_jit(_lazywhere, static_argnums=(2, 4), static_argnames=("f", "f2"))
```

Comment:
If we apply enough rigor in our patterns, we could search for `# lazy_jax_jit` (with initial hash) to spot all cases where the jit doesn't work yet.